config/config.go

```diff
@@ -464,7 +464,7 @@ func InitClient() error {
 		}
 		break
 	}
-	viper.Set("MinimumDownloadSpeed", downloadLimit)
+	viper.Set("Client.MinimumDownloadSpeed", downloadLimit)
```

Comment:
I think some people actually tweak this.  Can you insert backward compatibility code so the value of `MinimumDownloadSpeed`, if set, is used as the default for `Client.MinimumDownloadSpeed` (that is, an explicit `Client.MinimumDownloadSpeed` supersedes everything else)?